### PR TITLE
copyedits to LB4 docs from BLS

### PR DESCRIPTION
**Pull request by bschrammIBM**
## Checklist

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: docs/site/todo-tutorial-juggler.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+summary: LoopBack 4 Todo Application Tutorial - Adding juggler bridge
 ---
 
-### Adding Legacy Juggler
+### Adding Juggler Bridge
 
-The Legacy Juggler is a "bridge" between the existing
+The Juggler Bridge is a "bridge" between the existing
 [loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler)
 and the new LoopBack 4 architecture. It provides the capabilities required to
```
Comment (virkt25) [inline docs/site/todo-tutorial-juggler.md]: Weren't you also planning on removing the word `new` here?

Comment (bschrammIBM) [reply docs/site/todo-tutorial-juggler.md]: yes will do that

File: docs/site/sidebars/lb4_sidebar.yml [hunk 2]
```diff
@@ -29,11 +29,7 @@ children:
       url: todo-tutorial-scaffolding.html
       output: 'web, pdf'
 
-    - title: 'Adding legacy juggler'
```
Comment (dhmlau) [inline docs/site/sidebars/lb4_sidebar.yml]: so we're removing this from the sidebar?

Comment (virkt25) [reply docs/site/sidebars/lb4_sidebar.yml]: Yep. The page is about adding `RepositoryMixin` to the Application but apps generated with `lb4` now have that as part of the template. I guess the page itself should be deleted. I can do that in a follow up PR.

**Comment by virkt25** [comment]
> Just a question, by removing the "adding juggler bridge" from tutorial, do we still need to keep the page?

Yes, I'll make a follow up PR to delete the page. Leaving this one to be just @bschrammIBM's edits.

**Pull request merged**